docs/releasenotes/unreleased/fixes.2.rst

```diff
@@ -0,0 +1,12 @@
+Global variables used in the loop before assigning variable (#868)
+------------------------------------------------------------------
+
+The variable rules (such as W0919 ``unused-argument`` or I0920 ``unused-variable``) were not aware of the recursive
```

Comment:
I think that in the fix we should always include a description what's the result of the fix. You mention the problem that existed, but it should also state how it works now. Or at least that "it's now fixed".